packages/core/src/Stores/ui-store.js

```diff
@@ -16,6 +16,12 @@ export default class UIStore extends BaseStore {
     @observable is_positions_drawer_on = false;
     @observable is_reports_visible = false;
     @observable is_cashier_visible = false;
+    @observable is_history_tab_active = false;
+
+    // Take profit, Stop loss & Deal cancellation checkbox
+    @observable should_show_take_profit_warning = true;
+    @observable should_show_stop_loss_warning = true;
+    @observable should_show_deal_cancellation_warning = true;
```

Comment:
The UI store needs a clean-up. The observables introduced here will at some point need to be moved away from a common store and into something more DTrader specific (like Trade Store or a new one called TradeUI store).

If you can move these to the Trade store here, please do, otherwise, please leave a TODO with the tag `[cleanup-ui-store]`.

Reply:
sure, will move these to trade store instead